decidim-core/app/validators/password_validator.rb

```diff
@@ -96,10 +96,11 @@ def email_included_in_password?
   end
 
   def domain_included_in_password?
-    return false unless record&.current_organization&.host
-    return true if value.include?(record.current_organization.host)
+    organization = record.respond_to?(:current_organization) ? record.current_organization : record.organization
```

Comment:
It won't affect these two cases where this is **actually** used but what if `record` doesn't respond to `#organization`?

Also below this, what if `organization` or `current_organization` is `nil`?

Reply:
I improved this guard clause